src/ipyniivue_experimental/_options_mixin.py

```diff
@@ -0,0 +1,457 @@
+# This file is automatically generated by scripts/generate_options_mixin.py
+# Do not edit this file directly
```

Comment:
this file is automatically generated with getters/setters for the front-end properties.